ibmcloud/README.md

```diff
@@ -32,18 +32,20 @@ In addition to this you will need to install [`jq`](https://stedolan.github.io/j
 > $ sudo apt-get install jq
 > ```
 
-## Build and upload a peer pod VM image to IBM Cloud
+If you plan to run the end-2-end test framework you will also require [go](https://go.dev/doc/install) and `make` to be installed.
```

Comment:
go and make are needed for the documentation deploy process as well as the e2e tests, so I don't think this is optional, but is just a pre-req of this doc now?